support-frontend/assets/components/checkoutForm/checkoutForm.jsx

```diff
@@ -36,7 +36,7 @@ the top level form itself
 */
 
 type FormPropTypes = {
-  children: ChildrenArray<Element<typeof FormSection>>,
+  children: ChildrenArray<Element<any>>,
```

Comment:
This change is needed to allow us to include `FormSections` inside other components as we do with `PaymentMethodSelector`